components/selectors/context.rs

```diff
@@ -189,15 +189,6 @@ where
         }
     }
 
-    /// Override the quirks mode we're matching against.
-    ///
-    /// FIXME(emilio): This is a hack for XBL quirks-mode mismatches.
-    #[inline]
-    pub fn set_quirks_mode(&mut self, quirks_mode: QuirksMode) {
```

Comment:
Can't remove this; it's used by Gecko.

Reply:
Thanks for pointing this out, I'll leave it inside. The tool isn't perfect, it sometimes can't even detect when something is used by servo itself. You have to verify its output manually for false positives. I've ran some of the stuff through searchfox but I guessed I missed this one.

Reply:
Oh, that's a good point about SearchFox. You may actually be right that this is unused, but I would want @emilio to verify that.

Reply:
Yeah, this is used: https://searchfox.org/mozilla-central/rev/f8b11433159cbc9cc80500b3e579d767473fa539/servo/ports/geckolib/glue.rs#3734